File: parse_readme.js
const fs = require('fs');
const html = fs.readFileSync('readme.html', 'utf8');
const match = html.match(/id="hub-me" type="application\/json" data-json="(.+?)"/);
if (match) {
   // wait, hub-me has search metaData, but the real data is in a script block.
}

File: parse_readme.py
import json, re, sys
html = sys.stdin.read()
match = re.search(r'id="ssr-end">.*?<script id="__LOADABLE_REQUIRED_CHUNKS__".*?</script>.*?(<script.*?>.*?</script>)', html, re.DOTALL)
if match:
    # Actually wait, there is a better way. Let's look for "props":{"page":{"title":"Get Token"
    pass
